### ISSUE TITLE
Player skin selection for CS:GO completely gone after update

### ISSUE THREAD

**Issue by botarnold** [dvarnai/store-plugin#11]
After the 2/3/2016 update, I was unable to find the section in the store menu for player skins while others are unaffected? Note that this is player skins not weapon skins.

**Comment by Jose76Tron** [comment]
Yeah doesn't work anymore, idk why, and the owner of this plugin dissapeared

**Comment by Bara20** [comment]
Cecause he's too busy for the source kids and scammer. ;)
